viewer/packages/webpack.config.js

```diff
@@ -30,6 +30,10 @@ module.exports = env => {
       }
     },
 
+    node: {
+      fs: 'empty'
+    },
+
```

Comment:
The draco decoder has a dependency to the 'fs' (file system package in Nodejs) since it has some API calls where it can take in a path to a file to decode. This is unused by us so it does not cause any issues, but to build it needs the dependency, which I am basically polyfilling here.